{
  "Header": {
    "Original": "CVE-2024-1141 (A vulnerability was found in python-glance-store. The issue occurs whe ...)",
    "ID": "CVE-2024-1141",
    "Description": "(A vulnerability was found in python-glance-store. The issue occurs whe ...)"
  },
  "Annotations": [
    {
      "Original": "- python-glance-store \u003cunfixed\u003e",
      "Type": "package",
      "Package": "python-glance-store",
      "Kind": "unfixed"
    },
    {
      "Original": "[bookworm] - python-glance-store \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bookworm",
      "Package": "python-glance-store",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[bullseye] - python-glance-store \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bullseye",
      "Package": "python-glance-store",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2258836",
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=2258836"
    },
    {
      "Original": "TODO: check, missing details in RHBZ#2258836",
      "Type": "TODO",
      "Description": "check, missing details in RHBZ#2258836"
    }
  ]
}